source/components/JsxParser.js

```diff
@@ -79,7 +79,10 @@ export default class JsxParser extends Component {
       case 'Literal':
         return expression.value
       case 'MemberExpression':
-        return (this.parseExpression(expression.object) || {})[expression.property.name]
+        const thisObj = this.parseExpression(expression.object) || {}
+        const member = (thisObj)[expression.property.name]
+        if (typeof member === 'function') return member.bind(thisObj)
+        return member
```

Comment:
Wow... I really did not expect this kind of support to be so simple/elegant. That's fantastic. :)